### PR TITLE
add `.fail()` assertion - fixes #2

### LINKED ISSUE #2: Add `.fail` too?
Since we have `.pass()`.

Looks like tape has both: https://github.com/substack/tape

### PR DESCRIPTION
**Pull request by tunnckoCore**
#2

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -57,6 +67,7 @@ test('should not throw an error', function (t) {
 
 test('should not be an error', function (t) {
 	claim.error(false);
+	claim.error(null);
```
Comment (sindresorhus) [inline test.js:70]: Not relevant to this PR.

File: test.js [hunk 2]
```diff
@@ -7,6 +8,15 @@ test('.pass()', function (t) {
 	t.end();
 });
 
+test('.fail()', function (t) {
+	t.plan(1)
+	assert.throws(function () {
```
Comment (sindresorhus) [inline test.js:13]: AVA supports `throws` now, just bump the dependency.

File: test.js [hunk 3]
```diff
@@ -7,6 +8,15 @@ test('.pass()', function (t) {
 	t.end();
 });
 
+test('.fail()', function (t) {
+	t.plan(1)
```
Comment (sindresorhus) [inline test.js:12]: No point in using `.plan` for sync tests.

File: test.js [hunk 4]
```diff
@@ -7,6 +8,15 @@ test('.pass()', function (t) {
...
+	assert.throws(function () {
+		t.assert(true);
```
Comment (sindresorhus) [inline test.js:14]: No point in asserting inside a `throws` assertion.

Comment (tunnckoCore) [inline test.js:14]: Okey, how you will validate that it is thrown correctly then?

Comment (sindresorhus) [inline test.js:14]: That exactly what `assert.throws` does.

**Comment by sindresorhus** [comment]
ping @tunnckoCore

**Comment by tunnckoCore** [comment]
Oh yes, sorry. I'll come to it tomorrow.

**Pull request closed without merging**